roles/storage-cinder/README.md

```diff
@@ -0,0 +1,48 @@
+# Ansible Meta Role to Deploy Standalone Cinder
+
+This role aggregates Cinder, RabbitMQ and MariaDB to deploy Standalone
+Cinder with no authentication (noauth). 
+
+MariaDB is deployed on a node with label=mariadb. MariaDB uses hostPath
+for storage. 
+
+### Role Variables
+| variable | default |
```

Comment:
They have recently switched to a new standard tabular format for documenting the variables.  Please adopt it here.

Reply:
This change has been made.